### ISSUE TITLE
JAWs not reading toast notification message

### ISSUE THREAD

**Issue by WiproCRMAccessibility** [FreedomScientific/VFO-standards-support#186]
### Summary

> When a toast notification shown JAWs is not reading the notification message
> 
> Example:
> 1. Go to [Toast Demo](https://codepen.io/anon/pen/EMzraP)
> 2. Click on any button
> 3. toast notification will appear at the bottom of the page

### Expected result

> JAWs should read the toast notification message when it is shown.

### Actual result

> JAWs is not reading the notification message when a toast notification appears.

#### JAWS version and build number
Version 2018.1805.33 ILM
 
#### Operating System and version
Windows 10

#### Browser and version:
Chrome Version 72.0.3626.121 (Official Build) (64-bit)
IE11 version 11.379.17763.0
Microsoft Edge 44.17763.1.0

**Comment by scottaohara** [comment]
Hi @WiproCRMAccessibility 

This is not a JAWS bug your experiencing.  Rather it's an issue with how the toast has been implemented.  A live region must exist in the DOM for screen readers (not just JAWS, this doesn't work with VoiceOver either) to be aware of it.  Content can then be populated into the live region and it will be announced.

If you revise your code to have an empty `<div aria-live="polite"></div>` in the document by default, and then populate the toast container / text into it, you will likely have better results.

**Comment by JAWS-test** [comment]
Meanwhile it works with Firefox and Chrome. This means that even if the implementation is not correct, JAWS 2019 will output the messages.
